explorer/src/utils/index.tsx

```diff
@@ -115,3 +115,11 @@ export function camelToTitleCase(str: string): string {
   const result = str.replace(/([A-Z])/g, " $1");
   return result.charAt(0).toUpperCase() + result.slice(1);
 }
+
+export function formatDollarValue(value: number, decimals: number): string {
+  return "$" + Number(value.toFixed(decimals)).toLocaleString("en-US");
```

Comment:
use browser Intl.NumberFormat to format as USD

Reply:
Done